https://issues.redhat.com/browse/ACM-9870 FIPS updates (#6177)
open-cluster-management/rhacm-docs: release_notes/fips_readiness.adoc

[#fips-readiness]
= FIPS readiness

{product-title} is designed for FIPS. When running on {ocp} in FIPS mode, {ocp-short} uses the Red Hat Enterprise Linux cryptographic libraries submitted to NIST for FIPS Validation on only the architectures that are supported by {ocp-short}. For more information about the NIST validation program, see link:https://csrc.nist.gov/Projects/cryptographic-module-validation-program/validated-modules[Cryptographic Module Validation Program]. For the latest NIST status for the individual versions of the RHEL cryptographic libraries submitted for validation, see link:https://access.redhat.com/articles/2918071#fips-140-2-and-fips-140-3-2[Compliance Activities and Government Standards]. 

> If you plan to manage clusters with FIPS enabled, you must install {product-title-short} on an {ocp-short} cluster configured to operate in FIPS mode. The hub cluster must be in FIPS mode because cryptography that is created on the hub cluster is stored on provisioned managed clusters. 

Reviewer: ```suggestion
If you plan to manage clusters with FIPS enabled, you must install {product-title-short} on an {ocp-short} cluster configured to operate in FIPS mode. The hub cluster must be in FIPS mode because cryptography that is created on the hub cluster is used on managed clusters. 
```

> To enable FIPS mode on your managed clusters, set the following parameter when you provision your {ocp-short} managed cluster: `fips: true`. You cannot enable FIPS after you provision your cluster. For more information, see {ocp-short} documentation, link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html-single/installing/index#installing-preparing-security[Do you need extra security for your cluster?]

Reviewer 2: @dockerymick 2 things:

For more information, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html-single/installing/index#installing-preparing-security[Do you need extra security for your cluster?] in the {ocp-short} documentation?

Reviewer 2: Can we merge a bit here for clarity and conciseness:

...set `fips: true` when you provision your {ocp-short} managed cluster.

Reviewer 2: Also, can we point to 4.13 here or does it not exist for 4.13?

> To enable FIPS mode on your managed clusters, set the following parameter when you provision your {ocp-short} managed cluster: `fips: true`. You cannot enable FIPS after you provision your cluster. For more information, see {ocp-short} documentation, link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html-single/installing/index#installing-preparing-security[Do you need extra security for your cluster?]

Author: Yea it exists and to make a suggestion, select the "Add a suggestion" icon. it is next to the Preview tab of the comment box. 

<img width="484" alt="Screenshot 2024-03-04 at 2 38 45 PM" src="https://github.com/stolostron/rhacm-docs/assets/60616885/812b9b75-33cf-472e-912d-3341fa3fec62">

```suggestion
To enable FIPS mode on your managed clusters, set `fips: true` when you provision your {ocp-short} managed cluster. You cannot enable FIPS after you provision your cluster. For more information, see {ocp-short} documentation, link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/installing/index#installing-preparing-security[Do you need extra security for your cluster?]
```

[#fips-readiness]
= FIPS readiness

{product-title} is designed for FIPS. When running on {ocp} in FIPS mode, {ocp-short} uses the Red Hat Enterprise Linux cryptographic libraries submitted to NIST for FIPS Validation on only the architectures supported by {ocp-short}. For more information about the NIST validation program, see link:https://csrc.nist.gov/Projects/cryptographic-module-validation-program/validated-modules[Cryptographic Module Validation Program]. For the latest NIST status for the individual versions of the RHEL cryptographic libraries submitted for validation, see link:https://access.redhat.com/articles/2918071#fips-140-2-and-fips-140-3-2[Compliance Activities and Government Standards]. 

> If you plan to manage clusters with FIPS enabled, you must install {product-title-short} on an {ocp-short} cluster configured to operate in FIPS mode. You must also have a FIPS-ready hub cluster because cryptography that is created on the hub cluster is stored on the managed cluster. To enable FIPS mode on your, set the following parameter when you provision your {ocp-short} managed cluster: `fips: true`. You cannot enable FIPS after you provision your cluster. For more information, see {ocp-short} documentation, link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html-single/installing/index#installing-preparing-security[Do you need extra security for your cluster?]

Reviewer: Some aspects of this are not clear.  The ACM Hub needs to be installed on a cluster that has FIPS enabled is one point.  It's a second point to mention provisioning managed clusters and enabling FIPS for those.
```suggestion
If you plan to manage clusters with FIPS enabled, you must install {product-title-short} on an {ocp-short} cluster configured to operate in FIPS mode. The hub cluster must be in FIPS mode because cryptography that is created on the hub cluster is stored on provisioned managed clusters. 

To enable FIPS mode on your managed clusters, set the following parameter when you provision your {ocp-short} managed cluster: `fips: true`. You cannot enable FIPS after you provision your cluster. For more information, see {ocp-short} documentation, link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html-single/installing/index#installing-preparing-security[Do you need extra security for your cluster?]
```
